README.rst

```diff
@@ -164,11 +164,11 @@ The user typically uses the output files to produce final plots. For example,
 running ``plot.C`` output macro on the ``AnalysisResults.root`` output file will
 permit to visualise the pt distribution of the accepted events:
 
-.. figure:: https://raw.githubusercontent.com/reanahub/reana-demo-alice-pt-analysis/blob/master/docs/plot_pt.png
+.. figure:: https://github.com/reanahub/reana-demo-alice-pt-analysis/blob/master/docs/plot_pt.png
```

Comment:
We need to use "raw" to access the figure blob itself. The correct links are to raw.githubusercontent.com but without "/blob/" part of the URL. See for example:

```console
$ grep figure ../reana-demo-atlas-recast/README.rst
.. figure:: https://raw.githubusercontent.com/reanahub/reana-demo-atlas-recast/master/docs/workflow.png
.. figure:: https://raw.githubusercontent.com/reanahub/reana-demo-atlas-recast/master/statanalysis/example_results/pre.png
.. figure:: https://raw.githubusercontent.com/reanahub/reana-demo-atlas-recast/master/statanalysis/example_results/post.png
.. figure:: https://raw.githubusercontent.com/reanahub/reana-demo-atlas-recast/master/statanalysis/example_results/limit.png
```

Reply:
Fixed